packages/core/src/components/slider/coreSlider.tsx

```diff
@@ -22,7 +23,7 @@ export interface ICoreSliderProps extends IProps {
     disabled?: boolean;
 
     /**
-     * Increment between successive labels.
+     * Increment between successive labels. Must be non-zero.
```

Comment:
negative label step crashes the browser (infinite loop again) but step size just reverse the arrow keys 😄